controllers/operands/kubevirt.go

```diff
@@ -294,7 +295,71 @@ func hcWorkloadUpdateStrategyToKv(hcObject *hcov1beta1.HyperConvergedWorkloadUpd
 	return kvObject
 }
 
-func getKVConfig(hc *hcov1beta1.HyperConverged) (*kubevirtcorev1.KubeVirtConfiguration, error) {
+func getRatesFromConfigMap(configMap *corev1.ConfigMap) (*kubevirtcorev1.ReloadableComponentConfiguration, error) {
+
+	qps, ok := configMap.Data["qps"]
+
+	if !ok {
+		return nil, errors.New("qps parameter not found in configmap " + configMap.Name)
+	}
+
+	qpsConverted, err := strconv.ParseFloat(qps, 32)
+
+	if err != nil {
+		return nil, err
+	}
+
+	burst, ok := configMap.Data["burst"]
+	if !ok {
+		return nil, errors.New("burst parameter not found in configmap" + configMap.Name)
+	}
+
+	burstConverted, err := strconv.Atoi(burst)
+
+	if err != nil {
+		return nil, err
+	}
+
+	return &kubevirtcorev1.ReloadableComponentConfiguration{
+		RestClient: &kubevirtcorev1.RESTClientConfiguration{
+			RateLimiter: &kubevirtcorev1.RateLimiter{
+				TokenBucketRateLimiter: &kubevirtcorev1.TokenBucketRateLimiter{
+					QPS:   float32(qpsConverted),
+					Burst: burstConverted,
+				},
+			},
+		},
+	}, nil
+
+}
+
+func hcoTuning2Kv(hcTuning *hcov1beta1.Tuning, hcNamespace string) (*kubevirtcorev1.ReloadableComponentConfiguration, error) {
+	var kvReloadable *kubevirtcorev1.ReloadableComponentConfiguration
+	if hcTuning != nil {
+		if hcTuning.Policy != nil {
+			if hcTuning.Policy.Name != "" {
```

Comment:
if hcTuning.Policy.Name == "configmap" (please use a const in api/v1beta1/hyperconverged_types.go)

Reply:
Done.